src/main/groovy/org/mockito/release/notes/model/Commit.java

```diff
@@ -8,6 +8,11 @@
 public interface Commit {
 
     /**
+     * Commit identifier. For git it would be 'hash' (SHA-1)
+     */
+    String getCommitId();
```

Comment:
commit id is not really needed at the moment? I assume you added it for completeness?